linux/jdk/debian/src/main/packaging/microsoft/21/debian/rules

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/make -f
+
+pkg_name = msopenjdk-21
+priority = 1911
```

Comment:
Do we want to change the default priority of this package? I am not sure about this.
```suggestion
priority = 2111
```

Reply:
I'm ok with this change, although we may need to change it again when we see what the community agrees on